utoipa-axum/src/router.rs

```diff
@@ -253,8 +253,15 @@ where
             })
         };
 
-        // add current paths to the OpenApi
-        self.1.paths.paths.extend(paths.paths.clone());
+        // add or merge current paths to the OpenApi
+        for (path, item) in paths.paths.clone() {
```

Comment:
I think we can do this without cloning as well. The `clone` there previously has been just forgotten there most likely.